### DISCUSSION TITLE
How to translate sidebar text.

### DISCUSSION THREAD

**Discussion by Pascualsolutions** [filamentphp/filament#13050] [General]
Dear how to translate on my language text from sidebar.I translate Administration text but my resource cant find way. I atached prntscr.
Thank you
Alex

![Screenshot from 2024-06-01 09-59-58](https://github.com/filamentphp/filament/assets/75854186/1a198215-ec71-4505-b470-7bd315bce888)

**Comment by HamidGj** [comment]
Add the following two methods to your filament-resource class and change the 'User' , 'Users' to the desired resource name, and add it to your translation file (lang /*.json) or return your translation directly.
```
public static function getModelLabel(): string
{
    return __('User');
}
public static function getPluralModelLabel(): string
{
    return __('Users');
}
```
